annotate/module.ily

```diff
@@ -74,7 +74,8 @@
       (props (context-mod->props properties))
       ;; retrieve a pair with containing directory and input file
       (input-file (string-split (car (ly:input-file-line-char-column (*location*))) #\/ ))
-      (ctx (list-tail input-file (- (length input-file) 2)))
+      (ctx (list-tail input-file (- (length input-file)
+                                    (if (> (length input-file) 1) 2 1))))
```

Comment:
Ok I see what the whole thing was intended to do in the first place, so *technically* the wrong thing to do was to be in the file's directory like I was. Without applying the patch as is, `lilypond ... ``pwd``/min.ly` (excuse the double backquotes - I couldn't see how to highlight that chunk otherwise) could be a viable workaround. I'll try that out later (without the changes).

This doesn't stop any other things at the moment really, so don't go out of your way to worry about it.